selftests/functional/test_output.py

```diff
@@ -391,11 +379,11 @@ def test_redirect_output(self):
     @unittest.skipIf(perl_tap_parser_uncapable(),
                      "Uncapable of using Perl TAP::Parser library")
     def test_tap_parser(self):
-        perl_script = script.TemporaryScript("tap_parser.pl",
-                                             PERL_TAP_PARSER_SNIPPET)
-        perl_script.save()
         os.chdir(basedir)
-        process.run("perl %s" % perl_script)
+        result = process.run(os.path.join("selftests", ".scripts",
```

Comment:
I think we have discussed the idea of putting those snippets out of the unittests and into actual files to "help manual reproduction", but AFAIK we decided to keep them close(r) to their use, that is, inside the unittests.  Also, I don't get why we'd use a hidden directory for that.